pkgs/by-name/vo/volatility2-bin/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  stdenv,
+  fetchzip,
+  lib,
+  autoPatchelfHook,
+  libz,
+  darwin,
+}:
+let
+  suffix =
+    {
+      x86_64-linux = "lin64_standalone";
+      x86_64-darwin = "mac64_standalone";
+    }
+    .${stdenv.hostPlatform.system};
+  hash =
+    {
+      x86_64-linux = "sha256-ucG6oR4gBRUjMmHRr9QNenc04ENvwLvyCzSAqIoAiwM=";
+      x86_64-darwin = "sha256-BObRSSGUra1y/oo3ZFfIGi2PdHDX2gZy315x7R9DQPk=";
+    }
+    .${stdenv.hostPlatform.system};
+in
+stdenv.mkDerivation {
+  pname = "volatility2-bin";
+  version = "2.6.1";
+
+  src = fetchzip {
+    url = "https://github.com/volatilityfoundation/volatility/releases/download/2.6.1/volatility_2.6_${suffix}.zip";
+    stripRoot = true;
+    inherit hash;
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  buildInputs =
+    [
+      libz
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.Libsystem
+    ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  installPhase = ''
+    install $src/volatility_2.6_${suffix} -m 555 -p -D -T $out/bin/volatility2
+    ln -s $out/bin/volatility2 $out/bin/vol2
+    chmod 555 $out/bin/vol2
+  '';
+
+  meta = with lib; {
+    homepage = "https://volatilityfoundation.org/";
+    mainProgram = "volatility2";
+    description = "An advanced memory forensics framework";
+    platforms = [
+      "x86_64-linux"
+      "x86_64-darwin"
+    ];
+    maintainers = with lib.maintainers; [ ivyfanchiang ];
+    license = lib.licenses.gpl2;
```

Comment:
Add `sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];` and `changelog = "https://github.com/volatilityfoundation/volatility/releases/tag/${finalAttrs.version}";`